### PR TITLE
Add npm install instructions to readme

### PR DESCRIPTION
**Pull request by samuelli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ For browsers that need it, there are also some minor polyfills included:
 - [`URL constructor`](https://github.com/webcomponents/URL)
 
 ## How to use
+### Install polyfills
+```bash
+npm install @webcomponents/webcomponentsjs
```
Comment (justinfagnani) [inline README.md]: Right now this requires `npm install @webcomponents/webcomponentsjs@next`. Should we do that until it's tagged as `latest`?

Comment (justinfagnani) [reply README.md]: Also, for elements, this should have the `-D` flag...

Comment (samuelli) [reply README.md]: This is for the v2 branch, which I think won't be merged to master till this is published next week.

**Comment by TimvdLippe** [comment]
Oh ugh, this was done to the old `v2` branch, but it should have been master... @samuelli could you open another PR to master instead?

**Comment by samuelli** [comment]
> Oh ugh, this was done to the old v2 branch, but it should have been master... @samuelli could you open another PR to master instead?

Yeah, I made this PR when v2 was a thing :(

**Pull request merged**